### PR TITLE
Added typings to package.json

### PR DESCRIPTION
**Pull request by StefanRetief**
Adding the typings field allows the typescript server to find the typescript types for this package.

### CODE REVIEW

**Review by calebmshafer — APPROVED** [review]

**Comment by calebmshafer** [comment]
@StefanRetief are you looking to backport this to 2.19.x?

**Review by pmconne — APPROVED** [review]

**Comment by pmconne** [comment]
@Mergifyio update

**Comment by StefanRetief** [comment]
> @StefanRetief are you looking to backport this to 2.19.x?

@calebmshafer yes, that would be preferable

File: core/markup/package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "3.0.0-dev.22",
   "description": "iModel.js markup package",
   "main": "lib/imodeljs-markup.js",
+  "typings": "lib/imodeljs-markup",
```
Comment (aruniverse) [inline core/markup/package.json:6]: should this be lib/imodeljs-markup.d.ts

Comment (calebmshafer) [reply core/markup/package.json:6]: It doesn't necessarily need to have the `.d.ts` extension. Typescript should pick it up either way.

**Pull request merged**